Source/Project64-core/N64System/Mips/FlashRam.cpp

```diff
@@ -29,21 +29,21 @@ CFlashram::~CFlashram()
 
 void CFlashram::DmaFromFlashram(uint8_t * dest, int32_t StartOffset, int32_t len)
 {
-    uint8_t FlipBuffer[0x10000];
+    uint8_t FlipBuffer[1 << 16];
```

Comment:
I prefer 0x10000 over 1 << 16. It might be just me but at a quick looking I have no idea how much is 1 << 16. I have to spend time thinking and trying to calculate it to work out the size, instead of my attention of looking at the code.

The rest looks good tho